zero/zero_mq/queue_device/broker.py

```diff
@@ -13,7 +13,7 @@ def __init__(self):
     def listen(self, address: str, channel: str) -> None:
         self.gateway.bind(f"{address}")
         self.backend.bind(f"{channel}")
-        logging.info(f"Starting server at {address}")
+        logging.info("Starting server at %s", address)
```

Comment:
f string is faster?

Reply:
https://github.com/Ananto30/zero/pull/34#discussion_r1160584800